packages/react-aria-components/test/AriaMenu.test-util.tsx

```diff
@@ -621,8 +621,6 @@ export const AriaMenuTests = ({renderers, setup, prefix}: AriaMenuTestProps) =>
           expect(submenu).toBeInTheDocument();
 
           await submenuUtil.selectOption({option: submenuUtil.options().filter(item => item.getAttribute('aria-haspopup') == null)[0]});
-          // TODO: not ideal, this runAllTimers is only needed for RSPv3, not RAC or S2
-          act(() => {jest.runAllTimers();});
```

Comment:
no longer need after a `selectOption` call due to the chain of waitFors, seems to work fine for various levels of open menus but will extensive testing to see if it is too brittle or not